From a6a62c61c20f142bb37de1fe65d3586842f0461b Mon Sep 17 00:00:00 2001
From: Pixelnarium <hello+pixelgithub@creepy-net.de>
Date: Sun, 19 Aug 2018 22:23:33 +0200
Subject: [PATCH] 64bit

---
 cmake/BuildParameters.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/BuildParameters.cmake b/cmake/BuildParameters.cmake
index 12f23d0b6..351af568e 100644
--- a/cmake/BuildParameters.cmake
+++ b/cmake/BuildParameters.cmake
@@ -173,7 +173,7 @@ include(TargetArch)
 target_architecture(PCSX2_TARGET_ARCHITECTURES)
 if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "x86_64" OR ${PCSX2_TARGET_ARCHITECTURES} MATCHES "i386")
 	if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "x86_64" AND (CMAKE_BUILD_TYPE MATCHES "Release" OR PACKAGE_MODE))
-		message(FATAL_ERROR "
+    message(WARNING "
         The code for ${PCSX2_TARGET_ARCHITECTURES} support is not ready yet.
         For now compile with -DCMAKE_TOOLCHAIN_FILE=cmake/linux-compiler-i386-multilib.cmake
         or with
-- 
2.18.0

